### PR TITLE
Use auto-loading and allow updates via Composer

### PR DESCRIPTION
**Pull request by Alkarex**
It looks like permissions to contribute to the PR https://github.com/FreshRSS/FreshRSS/pull/4403 are disabled, so I had to publish to another branch

### CODE REVIEW

File: lib/.gitignore [hunk 1]
```diff
@@ -1,6 +1,14 @@
 autoload.php
 composer.lock
 composer/
+marienfressinaud/lib_opml/.git/
```
Comment (Alkarex) [inline lib/.gitignore:4]: 😱 😉

File: README.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ and [Fever API](https://freshrss.github.io/FreshRSS/en/users/06_Fever_API.html)
 * [SimplePie](https://simplepie.org/)
 * [MINZ](https://github.com/marienfressinaud/MINZ)
 * [php-http-304](https://alexandre.alapetite.fr/doc-alex/php-http-304/)
-* [lib_opml](https://github.com/marienfressinaud/lib_opml)
+* [lib_opml](https://framagit.org/marienfressinaud/lib_opml)
```
Comment (Alkarex) [inline README.md:141]: Related https://github.com/FreshRSS/FreshRSS/pull/4414

File: lib/.gitignore (hunk 1 above)
Comment (marien-probesys) [reply lib/.gitignore:4]: Ah ah, I would prefer to have something like `libopml`, but I created my account on Packagist with my name, and I don't if and how I can change that without creating a different account :sweat_smile:

Comment (marien-probesys) [reply lib/.gitignore:4]: Is the `.git` folder coming with composer? :o

Comment (marienfressinaud) [reply lib/.gitignore:4]: Is the `.git` folder coming with composer ? :o

(previous comment removed, still fighting with my different GH accounts :sweat_smile:)

Comment (Alkarex) [reply lib/.gitignore:4]: Yes, it looks like the .git/ folder is included in your Composer archive

Comment (marienfressinaud) [reply lib/.gitignore:4]: I just added a `.gitattributes` file, it should be better with the next version :) https://framagit.org/marienfressinaud/lib_opml/-/blob/main/.gitattributes

Comment (Alkarex) [reply lib/.gitignore:4]: https://github.com/FreshRSS/FreshRSS/pull/4415 :-)

**Pull request merged**